app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    value = 6
+    number_of_parts = 2
+    assert sum(split_integer(value, number_of_parts)) == value
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    value = 6
+    number_of_parts = 2
+    assert len(split_integer(value, number_of_parts)) == number_of_parts
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    value = 8
+    number_of_parts = 1
+    assert split_integer(value, number_of_parts) == [8]
```

Comment:
Do not use extra variables in tests
```suggestion
    assert split_integer(8, 1) == [8]
```